lib/MusicBrainz/Server/Report/ReleasesWithCAANoTypes.pm

```diff
@@ -0,0 +1,48 @@
+package MusicBrainz::Server::Report::ReleasesWithCAANoTypes;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id,
+            row_number() OVER (ORDER BY musicbrainz_collate(ac.name), musicbrainz_collate(r.name))
+        FROM (
+            SELECT DISTINCT r.*
+            FROM release r
+            WHERE EXISTS (
+            SELECT 1 FROM cover_art_archive.cover_art WHERE cover_art.release = r.id
+            )
+            AND NOT EXISTS (
+            SELECT 1 FROM cover_art_archive.cover_art_type JOIN cover_art_archive.cover_art ON cover_art_type.id = cover_art.id WHERE cover_art.release = r.id
+        )
+        ) r
+        JOIN artist_credit ac ON r.artist_credit = ac.id
+    ";
+}
+
```

Comment:
To enforce the name of the added template file, the following subroutine should be inserted here:

``` perl
sub table { 'releases_with_caa_no_types' }
```

Otherwise, default table/template name would be `releases_with_c_a_a_no_types`.